### DISCUSSION TITLE
Can a plot win more than once?

### DISCUSSION THREAD

**Discussion by XIII39** [Chia-Network/chia-blockchain#6103] [Q&A]
When a plot wins, can it win again? Ever?
Wondering if I should be replacing plots that have won with new ones

**Comment by tiny0little** [comment]
I saw somewhere in the docs similar question, the answer was something like this - good quality k32 plot has over 30 proofs, even if it won once, there are still good chances for it win again (using other proofs)

**Reply by XIII39** [reply]
I thought that might be the case. I guess time will tell.
Thanks!

**Reply by FSys77** [reply]
In theory all blocks could be won by only one single plot - although highly unlikely :-D
(but in one parallel universe exactly this happens)

**Reply by XIII39** [reply]
@FSys77 A guy that lived down the road from me won the lottery twice in one year so who knows, right?

**Comment by rcamarda390** [comment]
In my mind, the question is once a plot wins, should it be removed and create a new one to replace it?

**Reply by XIII39** [reply]
I would hate to eventually have a HDD full of plots that will never win again.. I did read somewhere about larger farms deleting plots that previously won - I'll have to find the link to that article - but I think that was more related to privacy issues more than holding unwinnable plots. 

I think like @tiny0little mentioned, a good quality plot has the potential to win multiple times but in reality, only time will tell.
As we start to get more longevity in the data we will be able to see what the reality is of a single plot winning over again..
